devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -101,5 +144,323 @@ protected SpanByte SliceGenericBuffer(int start, int length)
 
             return _genericBuffer;
         }
+
+        /// <summary>
+        /// Draws a pixel on the screen.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="inverted">Indicates if color to be used turn the pixel on, or leave off.</param>
+        public void DrawPixel(int x, int y, bool inverted = true)
+        {
+            if ((x >= Width) || (y >= Height))
+            {
+                return;
+            }
+
+            // x specifies the column
+            int idx = x + (y / 8) * Width;
+
+            if (inverted)
+            {
+                _genericBuffer[idx] |= (byte)(1 << (y & 7));
+            }
+            else
+            {
+                _genericBuffer[idx] &= (byte)~(1 << (y & 7));
+            }
+        }
+
+        /// <summary>
+        /// Draws a horizontal line.
+        /// </summary>
+        /// <param name="x0">x coordinate starting of the line.</param>
+        /// <param name="y0">y coordinate starting of line.</param>
+        /// <param name="length">Line length.</param>
+        /// <param name="inverted">Turn the pixel on (true) or off (false).</param>
+        public void DrawHorizontalLine(int x0, int y0, int length, bool inverted = true)
+        {
+            for (var x = x0; (x - x0) < length; x++)
+            {
+                DrawPixel(x, y0, inverted);
+            }
+        }
+
+        /// <summary>
+        /// Draws a vertical line.
+        /// </summary>
+        /// <param name="x0">x coordinate starting of the line.</param>
+        /// <param name="y0">y coordinate starting of line.</param>
+        /// <param name="length">Line length.</param>
+        /// <param name="inverted">Turn the pixel on (true) or off (false).</param>
+        public void DrawVerticalLine(int x0, int y0, int length, bool inverted = true)
+        {
+            for (var y = y0; (y - y0) < length; y++)
+            {
+                DrawPixel(x0, y, inverted);
+            }
+        }
+
+        /// <summary>
+        /// Draws a rectangle that is solid/filled.
+        /// </summary>
+        /// <param name="x0">x coordinate starting of the top left.</param>
+        /// <param name="y0">y coordinate starting of the top left.</param>
+        /// <param name="width">Width of rectabgle in pixels.</param>
+        /// <param name="height">Height of rectangle in pixels</param>
+        /// <param name="inverted">Turn the pixel on (true) or off (false).</param>
+        public void DrawFilledRectangle(int x0, int y0, int width, int height, bool inverted = true)
+        {
+            // width--;
```

Comment:
I know I4m a bit picky, but what is the usage of this dead code? Please add a comment why you need to keep it or just remove it. Thanks!